app/fullService/api/buildGiftCode.ts

```diff
@@ -55,12 +56,8 @@ const buildGiftCode = async ({
 
   // TODO fix type, right now it just matches what the component is expecting
   const totalValueConfirmation = txProposal.outlayList
-    .map((outlay) => {
-      return BigInt(outlay.value);
-    })
-    .reduce((acc, cur) => {
-      return acc + cur;
-    });
+    .map((outlay) => BigInt(outlay.value))
+    .reduce((acc, cur) => acc + cur);
```

Comment:
Suggest including the initial value for .reduce() as a generally good practice

Reply:
Also, why not do everything in a single reduce() ? You can convert to BigInt and sum in the same statement.

Reply:
This wasn't my change, just fixed formatting. Happy to address but not sure if it's in scope for this PR?

Reply:
I suppose it could be, but I was only fixing some prettier issues here, not the logic. Might just be part of the refactor if it just works right now.

Reply:
Ideally, this value doesn't have to be calculated by the client. It should be in Full-Service. (I'll make it a ticket for that). Until then, we'll keep doing this.